.eslintrc.js

```diff
@@ -7,6 +7,10 @@ module.exports = {
     {
       files: ['*.ts'],
       extends: ['@metamask/eslint-config-typescript'],
+      rules: {
+        '@typescript-eslint/triple-slash-reference': 'off',
+        'spaced-comment': ['error', 'always', { markers: ['/'] }],
+      },
```

Comment:
For posterity: Needed for typescript `<reference />` directives. Someone with a better understanding of how declarations work is welcome to make these exceptions unnecessary.